components/form/form_validation.php

```diff
@@ -335,7 +335,7 @@ function max_length(int $count, string $msg = null)
 
     function min_length(int $count, string $msg = null)
     {
-        if ($this->should_test() && strlen($this->value) < $count) {
+        if ($this->should_test() && strlen($this->value ?? "") && strlen($this->value ?? "") < $count) {
```

Comment:
You can simplify this statement I think:
```suggestion
        if ($this->should_test() && strlen($this->value ?? "") < $count) {
```